templates/olm.enmasse.env

```diff
@@ -1,6 +1,6 @@
 # variables used in the olm yaml files
 
-VERSIONED_NAME=enmasse.${VERSION}.0
+VERSIONED_NAME=enmasse.${MAVEN_VERSION}
```

Comment:
As VERSIONED_NAME is used as the basis of the Operator's version number, don't we need to maintain a level of indirection to suit downstream builds?

Reply:
There is already a downstream patch to set a different value for this field. This prevents the need to change an additional line in templates/csv/enmasse.package.yaml